bundles/org.openhab.binding.exec/src/main/java/org/openhab/binding/exec/internal/handler/ExecHandler.java

```diff
@@ -58,6 +58,9 @@
  */
 @NonNullByDefault
 public class ExecHandler extends BaseThingHandler {
+
+    private static final String EXEC_HANDLER_THREADPOOL_NAME = "execBinding";
```

Comment:
Something minor, I don't know when the thread pool name is displayed.
We don't have a clear naming convention for thread pool (we have one for threads) but most of my time yesterday was to check how the different bindings named them. The most common is to use the binding ID. Your naming would add a new way of naming that is not yet used.
My suggestion would be to use binding ID (using the existing constant).
WDTY ?

Reply:
We did fix many, many thread names to match the OH-bindingId convention., there is probably more todo.
@lolodomo i’ll leave this to you, I’m fine with either one.

Reply:
Wouldn't it be preferable then that these *don't* have the same naming convention, otherwise we could end up with multiple threads with the same name..?